kernel/Kconfig

```diff
@@ -508,24 +508,16 @@ config SYS_CLOCK_TICKS_PER_SEC
 	help
 	  This option specifies the nominal frequency of the system clock in Hz.
 
-	  Depending on the choice made, an amount of possibly expensive math must
-	  occur when converting ticks to milliseconds and vice-versa. Some values
-	  are optimized, and yield significantly less math.
-
-	  The optimal values from a computational point-of-view are 1000, 500,
-	  250 and 125, since in these cases there is either no computation
-	  required, or it is all done via bit-shifting. These also give a
-	  granularity from 1ms to 8ms.
-
-	  Other good values are 100, 50, 25, 20 and 10. In this case, some math
-	  is required but is minimized. These are also values that necessitate a
-	  reduced number of clock interrupts per second, at the cost of
-	  granularity (10ms to 100ms).
-
-	  All other values require some extensive 64-bit math, and in some
-	  configurations even require calls to compiler built-in functions, and
-	  can require a non-trivial extra amount of stack space (e.g. around 80
-	  bytes on x86).
+	  For asynchronous timekeeping, the kernel defines a "ticks" concept.  A
```

Comment:
what used to be easy to read is now condensed in one paragraph. 
Also, "The choice of tick rate is configurable via :c:option:`CONFIG_SYS_CLOCK_TICKS_PER_SEC`" sound like redundant, because this is the same option we are documenting.

Reply:
Changed it to a 2 para and removed via :c:option:CONFIG_SYS_CLOCK_TICKS_PER_SEC